frontend/libs/studio-components/src/components/StudioTextResourcePicker/StudioTextResourcePicker.tsx

```diff
@@ -23,11 +24,16 @@ export const StudioTextResourcePicker = forwardRef<HTMLInputElement, StudioTextR
       [onValueChange],
     );
 
+    const selectedValues: string[] = useMemo(
+      () => (doesTextResourceExist(textResources, value) ? [value] : []),
```

Comment:
Should we move the ternary conditional to a function? 😄

Reply:
Good call, I'll fix!

Reply:
Fixed.